### ISSUE TITLE
Add Google Scholar meta tags

### ISSUE THREAD

**Issue by mjordan** [bondjimbond/islandora_metadata_extras#23]
We already have the option to embed DC `<meta>` tags in objects' HTML, but we should also provide the option to embed [tags used by Google Scholar](https://scholar.google.com/intl/en/scholar/inclusion.html#indexing), e.g.

```
<meta name="citation_author" content="Wong, Victoria" />
<meta name="citation_title" content="A South African Developmental State? The Need to Overcome Incapacity and Pursue Embedded Autonomy in the 21st Century; - AND - On the Fringes: Housing Policy, Urban Slums, and the Necessity for a New Direction in South Africa" />
<meta name="citation_publication_date" content="2016/08/08" />
<meta name="citation_pdf_url" content="http://summit.sfu.ca/system/files/iritems1/18128/etd10599_VWong.pdf" />
```

**Comment by bondjimbond** [comment]
Is this covered by Islandora Google Scholar? https://github.com/Islandora/islandora_scholar/tree/7.x/modules/islandora_google_scholar

**Comment by bondjimbond** [comment]
Just reminding you that this is open. If Islandora Google Scholar covers the use case, is its dependency on Islandora Scholar a roadblock?

**Comment by mjordan** [comment]
@bondjimbond The background to adding this feature to Metadata Extras is https://jira.duraspace.org/browse/ISLANDORA-2262. Our current IR has good GS integration and we want to keep it without having to use the entire Islandora Scholar suite when we migrate to Islandora. I never got around to auditing Islandora Google Scholar to see it's one of the Islandora  Scholar modules that can be easily decoupled from the suite. If it is, Metadata Extras need not replicate that functionality.

**Comment by bondjimbond** [comment]
It makes use of variables like `islandora_scholar_xpaths_title`, which are set in the parent module, so I'm guessing it's not easily decoupled. 

But then, if you want to replicate the functionality in Metadata Extras, you would ideally make the xpaths similarly configurable, which would add a lot of overhead to your config form.

Maybe it would be better suited as an option in [Islandora Social Metatags](https://github.com/Islandora-Labs/islandora_social_metatags)?

**Comment by mjordan** [comment]
Good idea. We have not reapplied our efforts to our postponed IR migration yet but will need to do so soon. We'll want Google Scholar on opening day though.
